src/02-common-principles.md

```diff
@@ -275,8 +274,12 @@ Tabular data MUST be saved as tab delimited values (`.tsv`) files, i.e., CSV
 files where commas are replaced by tabs. Tabs MUST be true tab characters and
 MUST NOT be a series of space characters. Each TSV file MUST start with a header
 line listing the names of all columns (with the exception of physiological and
-other continuous acquisition data - see below for details). Names MUST be
-separated with tabs. String values containing tabs MUST be escaped using double
+other continuous acquisition data - see [below for details](04-modality-specific-files/06-physiological-and-other-continuous-recordings.md)).
```

Comment:
Can we change the parenthetical to:

```
(with the exception of [physiological and other continuous
recordings](04-modality-specific-files/06-physiological-and-other-continuous-recordings.md)).
```

([Make link text meaningful](https://www.w3.org/WAI/tips/writing/#make-link-text-meaningful))